project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=1.2.8
+sbt.version=1.3.2
```

Comment:
please not that this would bump up the minimum required sbt version also to 1.3.x. A workaround is proposed here - https://github.com/sbt/sbt/issues/5049#issuecomment-528960415

Reply:
Should be fixed by the latest commit.